src/atoms/Layout/__tests__/layout.spec.js

```diff
@@ -1,22 +1,29 @@
 import React from 'react';
+import renderer from 'react-test-renderer';
 import { mount, shallow } from 'enzyme';
-import { expect } from 'chai';
 
 import Layout from 'atoms/Layout';
 import Col from 'atoms/Layout/Col';
 
 describe('<Layout />', () => {
 
-  it('renders', () => {
+  it('creates a Layout component', () => {
     const wrapper = mount(<Layout />);
 
-    expect(wrapper.type()).to.equal(Layout);
+    expect(wrapper.type()).toEqual(Layout);
+  });
+
+  it('renders correctly', () => {
+    const component = renderer.create(<Layout />).toJSON();
```

Comment:
just curious why we need this `renderer` here. you don't seem to use it for the other snapshot checks